src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -312,6 +302,29 @@ public class NonEmptyStringToCollapsedConverter : IValueConverter
         }
     }
 
+    /// <summary>
+    /// Determines what the Install button says on the Package Manager Search.
+    /// If the package is installed it says 'Installed', otherwise 'Install'.
+    /// </summary>
+    public class InstalledButtonTextConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter,
+            CultureInfo culture)
+        {
+            if (!(value is bool booleanValue)) return null;
+            
+            return booleanValue
+                ? Properties.Resources.PackageManagerInstalled
+                : Properties.Resources.PackageManagerInstall;
```

Comment:
use `Wpf.Properties.Resources` so you dont have to define these in DynamoCore projects at all :)

Reply:
Done!